lib/Runtime/Debug/TTEvents.h

```diff
@@ -474,6 +475,26 @@ namespace TTD
 
         void ExplicitLogWriteEntry_Emit(const EventLogEntry* evt, FileWriter* writer, ThreadContext* threadContext);
         void ExplicitLogWriteEntry_Parse(EventLogEntry* evt, ThreadContext* threadContext, FileReader* reader, UnlinkableSlabAllocator& alloc);
+
+        //A struct for when we explicitly write a log entry with innerloop ttd end-breakpoint 
+        struct TTDInnerLoopLogWriteEventLogEntry
+        {
+            //copies the data for a TTDebuggerSourceLocation that we can set when we start the replay session (and track the end of the execution)
+            TTD_LOG_PTR_ID SourceScriptLogId;
+            int64 ETime;  //-1 indicates an INVALID location
+            int64 FTime;  //-1 indicates any ftime is OK
+            int64 LTime;  //-1 indicates any ltime is OK
+
+            uint32 TopLevelBodyId;
+            uint32 FunctionLine;
+            uint32 FunctionColumn;
+
+            uint32 Line;
```

Comment:
some comments describing differences between `Line` & `FunctionLine` and `Column` & `FunctionColumn` would be helpful here.  Or better names that clarify distinction.